app/uploaders/facility_image_uploader.rb

```diff
@@ -0,0 +1,35 @@
+class FacilityImageUploader < CarrierWave::Uploader::Base
+
+  include CarrierWave::RMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
```

Comment:
maybe this also should be in the CarrierWave initializer and use also `fog` on production like the `CourtImagesUploader`?